apps/blog/content/posts/what-is-memebattle.en.mdx

```diff
@@ -7,8 +7,106 @@ tags: ['MemeBattle']
+><b>MemeBattle</b> – small community around few technical projects, where we studied new technologies and approaches. <br /> Also this is the team with which we participated in hackathons and other activities.
+
+## MemeBattle today
+
+The initial concept of an application centered around battle of memes is a thing of the past.
+
+At the moment MemeBattle is primarily an <b>open community</b> united by common projects.
+We are open to new ideas and projects, also we invite everyone to join us.
+
+There are few projects in development by MemeBattle today.
+
+### Ligretto
+
+<b>Ligretto</b> – dynamic online multiplayer game, where needed to put cards as soon as possible and outperform opponents with reaction and dexterity.
+
+url: [https://ligretto.app/](https://ligretto.app/)
+
+Play with friends anytime – one round takes only few minutes. Our colleagues spend a free time between meetings sometimes)
+
+### GameHub
+
+<b>GameHub</b> – online games service. Unfortunately there is only one game so far. But in the ideal future it will be the portal of online games in one ecosystem.
+
+url: [https://mems.fun/](https://mems.fun/)
+
+Our lives
+We are witnessing a profound transformation in people's lives. Whe world is becoming global and remote.
+In the past, we used to have the chance to effortlessly gather with friends or colleagues at a party or in the office, enjoy ourselves, and engage in board games.
+
+Nowadays, many companies are spread across the globe, making it more difficult to gather around one table.
+GameHub and games within it can facilitate spending time together by playing.
+
+### Blog
+
+This blog. It's not just MemeBattle team news, but
+
+There will be not only news from the MemeBattle team, but also various posts from everyone.
+First of all, would like to share technical discoveries and experience. For example, one of next posts planned as description of using variable fonts research in this blog.
+
+url: [https://blog.mems.fun/](https://blog.mems.fun/)
+
+### MemeBattle as PaaS
+
+All our projects deployed to self infrastructure on AWS. We have an established practices and solutions for fast services launch to production.
+
+### Education
+
+We continue to use our projects for education. New technologies, tools, approaches etc.
+
+> It's better to try something on real-like projects than read about it a thousand times.
+
+#### Experience
+
+Here, you can practice solving problems like on real-life projects. For example, [add localization system to Ligretto](https://github.com/MemeBattle/monorepo/issues/177), so that the language barrier doesn't spoil the gaming experience.
+Or add new features like [leaderboard in Ligretto](https://github.com/MemeBattle/monorepo/milestone/11).
+
+Also there are smaller tasks that help newcomers take their [first steps](https://github.com/MemeBattle/monorepo/labels/good%20first%20issue).
+
+The experience gained from our projects consistently assists me during interviews and in my work.
```

Comment:
The experience gained on our projects constantly helps me in interviews and in my work.
And since our repositories are open and projects are publicly accessible, every member of the community has tangible evidence of their experience.